app.js

```diff
@@ -2,17 +2,25 @@ const express = require('express');
 const app = express();
 const { getTopics } = require('./controllers/controller.topics')
 const { getEndpoints } = require('./controllers/controller.endpoints');
+const { getArticleId } = require('./controllers/controller.articles')
+
 const {
   invalidEndpoint,
   handleCustomErrors,
   handlePsqlErrors,
   handleServerErrors,
 } = require('./errors/errors-controller');
 
+app.use(express.json());
```

Comment:
You do not need express.json until a PATCH/POST endpoint is created please delete this